src/client/systems/NetworkingClient.cpp

```diff
@@ -12,7 +12,7 @@
 #include "../../common/util/game_settings.hpp"
 #include "../../common/util/network_util.hpp"
 
-const sf::Time CLIENT_TCP_TIMEOUT = sf::milliseconds(100);
+const sf::Time CLIENT_TCP_TIMEOUT = sf::milliseconds(300);
```

Comment:
this timeout might still be too low in a case where we have 500 ms latency, but we probably should just tell the player their connection is too fucked